library/influxdb

```diff
@@ -60,10 +60,18 @@ Tags: 2.0-alpine, 2.0.9-alpine
 Architectures: amd64, arm64v8
 Directory: influxdb/2.0/alpine
 
-Tags: 2.1, 2.1.1, latest
+Tags: 2.1, 2.1.1
```

Comment:
Are all these older versions still actively supported? (Especially some of those older 1.x series?)

For reference, removing tags here will remove them from the "Supported" section on the Hub readme (and will prevent us from spending cycles rebuilding them on the official build servers), but the tags will still be available to users who want them.  (See https://github.com/docker-library/official-images#library-definition-files for more detail on this.)

Reply:
Hey, I've removed the 1.7 tags. I'm having internal discussions. However, currently, everything else is still currently supported.

Reply:
Excellent, thank you for double checking! :heart: